<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-N3S3FJH" failures="1" tests="6" name="Modificacion PIN RT" time="31.92" errors="0" timestamp="2024-04-29T10:47:03 CLT">
  <testcase classname="valuesite.tests.ModificacionPinTestRT" name="vaciaCampo" time="1.522"/>
  <testcase classname="valuesite.tests.ModificacionPinTestRT" name="ingresarPin" time="0.456"/>
  <testcase classname="valuesite.tests.ModificacionPinTestRT" name="actualizaPin" time="0.812"/>
  <testcase classname="valuesite.tests.ModificacionPinTestRT" name="actualizaSinDatos" time="0.394"/>
  <testcase classname="valuesite.tests.ModificacionPinTestRT" name="actualizaConCaracteresEspeciales" time="2.667"/>
  <testcase classname="valuesite.tests.ModificacionPinTestRT" name="limpiaCampo" time="0.485">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at valuesite.tests.ModificacionPinTestRT.limpiaCampo(ModificacionPinTestRT.java:169)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:842)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- limpiaCampo -->
</testsuite> <!-- Modificacion PIN RT -->
